src/YoutubeAtomPlayer.tsx

```diff
@@ -76,6 +76,29 @@ const createOnStateChangeListener = (
     const player = event.target;
 
     if (event.data === YT.PlayerState.PLAYING) {
+        const youtubeIframes = document.querySelectorAll(
+            '[data-atom-type="youtube"]',
+        );
+
+        /**
+         * send a 'stopVideo' postMessage to all other YoutubeAtoms
+         */
+        Array.from(youtubeIframes).forEach((iframe) => {
+            if (iframe instanceof HTMLIFrameElement) {
+                // filtering out the current component playing video
+                if (iframe.id !== `youtube-video-${videoId}`) {
+                    iframe.contentWindow?.postMessage(
+                        JSON.stringify({
+                            event: 'command',
+                            func: 'stopVideo',
```

Comment:
This is more of a functional question but do we want to _stop_ the video? This would re-cue the video to the start?

Would pausing be a bit more useful to the user so that they can resume the other video where they left off if they want to?

Reply:
+1 on this. Pausing would, I think, be a better experience

Reply:
Thanks all for pointing this out, and thanks @joecowton1 for quickly making the PR for adding the pause functionality to the interface. 

I made the change and the play event handler is now pausing the video instead of stopping